app/views/insured/consumer_roles/help_paying_coverage.html.erb

```diff
@@ -17,9 +17,17 @@
 
             <div id="help_paying_coverage" class="focus_effect module">
 
-              <div class="row row-form-wrapper no-buffer">
+              <div class="row row-form-wrapper no-buffer <%= 'pb-1' if EnrollRegistry[:mainecare_cubcare_glossary].enabled? %>">
```

Comment:
It's better to use the feature_enabled? syntax but I'm checking to see the difference